python/lsst/ts/standardscripts/base_block_script.py

```diff
@@ -125,6 +141,12 @@ async def configure(self, config):
         self.program = getattr(config, "program", None)
         self.reason = getattr(config, "reason", None)
         self.test_case = getattr(config, "test_case", None)
+        if self.test_case is not None:
```

Comment:
The test_case functionality can be removed now, we do not need to leave this for the future.